app/app.html

```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8">
     <title>Hearmenow</title>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
We should style change the dialer to use our material style when we get the chance. Shouldn't be using bootstrap and md but this works for now.